data/recipes/gce_disk-export.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "gce_disk_export",
+    "short_description": "Export disk image from a GCP project to Google Cloud Storage.",
+    "description": "Creates a disk image from a Google compute persistence disk, compress the image and export it to Google Cloud Storage.\n\nThe exported image name is appended by .tar.gz.\n\nThe service account [PROJECT-NR]@cloudbuild.gserviceaccount.com in the source project must have the IAM role [Srorage Admin] on the destination storage bucket/folder.",
+    "modules": [{
+        "wants": [],
+        "name": "GoogleCloudDiskExport",
+        "args": {
+            "analysis_project_name": "@analysis_project_name",
+            "source_project_name": "@source_project_name",
+            "source_disk_name": "@source_disk_name",
+            "gcs_output_location": "@gcs_output_location",
+            "exported_disk_name": "@exported_disk_name"
+        }
+    }],
+    "args": [
+        ["--analysis_project_name", "Project where the disk image is created then exported. If not exist, source_project_name will be used.", null],
```

Comment:
please put optional args at the end. I don't think it makes any difference in the code, but it's easier to understand what parameters the recipe needs.